langjo-frontend/src/Components/Footer.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import "./Footer.css";
+
+const Footer = () => (
+  <footer className="footer">
+    <div className="d-flex flex-column flex-md-row p-4 gap-5 py-md-5 align-items-center justify-content-center">
+      <div
+        className="position-static d-flex flex-column flex-lg-row align-items-stretch
+        justify-content-start p-3 rounded-3 shadow-lg"
+      >
+        <nav className="">
+          <ul className="list-unstyled d-flex flex-column">
+            <li>
+              <div>
+                <p className="cookies">CONTACT US</p>
+                <small className="small">
+                  If you have a question about the Learn English Network, <br />{" "}
+                  fill out our Contact Form, and we will get back to you asap.
+                </small>
+              </div>
+            </li>
+          </ul>
+        </nav>
+      </div>
+      <div className="d-flex flex-column flex-md-row p-4 gap-5 py-md-5 align-items-center justify-content-center">
+        &nbsp; &nbsp;
+      </div>
+      <div
```

Comment:
Is it possible to center this div?  Then the logo colors should be reflected

Reply:
We need a proper logo for the site.